public/stylesheets/style.css

```diff
@@ -1,13 +1,45 @@
 body {
-  padding: 50px;
-  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+  font-family: 'Quicksand', sans-serif;
+  color: #3c5a71;
 }
 
 a {
-  color: #00B7FF;
+  color: inherit;
+}
+a:hover {
+  text-decoration: none;
+}
+
+.ccnavbar {
```

Comment:
I don't think this belongs in this file
What is this file?
Is it the base styles? If so, call it base.css